$OpenBSD: patch-configure,v 1.5 2011/03/06 11:43:38 sthen Exp $

XXX make it compile with gtk+2 2.22 (*_DISABLE_DEPRECATED)
graphics/ufraw to fix : ufraw_preview.c:4026: error: expected declaration specifiers or '...' before 'GtkNotebookPage'

--- configure.orig	Sat Feb 19 05:51:44 2011
+++ configure	Mon Feb 21 09:51:02 2011
@@ -5592,9 +5592,6 @@ $as_echo "#define G_DISABLE_DEPRECATED /**/" >>confdef
 $as_echo "#define GDK_DISABLE_DEPRECATED /**/" >>confdefs.h
 
 
-$as_echo "#define GTK_DISABLE_DEPRECATED /**/" >>confdefs.h
-
-
 $as_echo "#define GTK_DISABLE_SINGLE_INCLUDES /**/" >>confdefs.h
 
 
